x11/xorg-xorgproto/Portfile

```diff
@@ -3,13 +3,12 @@
 PortSystem       1.0
 
 name             xorg-xorgproto
-epoch            1
-version          2021.4
+version          2021.4.99.2
 revision         0
 
-checksums        rmd160  7003602d02df7bda5da22412f6a35d4e144d4a11 \
-                 sha256  0f5157030162844b398e7ce69b8bb967c2edb8064b0a9c9bb5517eb621459fbf \
-                 size    885414
+checksums       rmd160  557cd714e87645e67b584e62f1b09aa952c6a2f6 \
+                sha256  c38878053179c6f8bc2daeeeeb4710b5fbf0e46db5b3134aee4a1977ffb06e7a \
+                size    875274
```

Comment:
please align the checksums correctly

Reply:
Technically these lines are aligned correctly (to multiples of 4 spaces), unfortunately most of the rest of the portfile is not aligned correctly (using spaces not at multiples of 4 spaces, or using tabs). I'd recommend aligning these checksums to match the existing "wrong" indentation of most of the rest of the portfile, unless you plan to follow up with a commit that fixes all of the rest of the whitespace of the portfile.

Reply:
yes, I would prefer the update commit to match the origin indentation, and then in a dedicated commit we should fix the indentation in the port file.